asteroid/complex_nn.py

```diff
@@ -61,16 +61,19 @@ def as_torch_complex(x, asteroid_dim: int = -2):
         is_torchaudio_complex = transforms.is_torchaudio_complex(x)
         is_asteroid_complex = transforms.is_asteroid_complex(x, asteroid_dim)
         if is_torchaudio_complex and is_asteroid_complex:
-            raise ValueError(
+            warnings.warn(
                 f"Tensor of shape {x.shape} is both a valid Torchaudio-style and "
                 "Asteroid-style complex. PyTorch complex conversion is ambiguous."
+                "Assuming Torchaudio-style.",
+                RuntimeWarning,
             )
+            return torch.view_as_complex(x)
```

Comment:
I thought you'd like to debate that @jonashaag.
It's very unlikely that an asteroid tensor will have length 2 in it's last dimension, however very possible that a torchaudio tensor has even shape or dim -2. That's why we should default to torchaudio style IMO.

Reply:
Hmm... I'm not sure what's the advantage of handling this case. The idea of this generic conversion function was that if you write some code (say a library) that uses complex numbers, you can accept any type of complex representation. So you don't have to know the representation upfront.

If you know upfront in the library what type of tensor you have then you can do the conversion directly without this function.

If you don't know what you have in the library, then IMO the library should complain to its user that it got input that it didn't understand, which is what the current code does.

Reply:
I knew you would disagree hahah 
As we discussed. Let's remove this function entirely and add `to_torch_complex` is `asteroid-filterbanks`.